base_kanban_stage/README.rst

```diff
@@ -0,0 +1,108 @@
+.. image:: https://img.shields.io/badge/licence-lgpl--3-blue.svg
+   :target: http://www.gnu.org/licenses/LGPL-3.0-standalone.html
+   :alt: License: LGPL-3
+
+======================
+Kanban - Stage Support
+======================
+
+This module provides a stage model compatible with Kanban views and the 
+standard views needed to manage these stages. It also provides the 
+``base.kanban.abstract`` model, which can be inherited to add support for 
+Kanban views with stages to any other model. Lastly, it includes a base Kanban 
+view that can be extended as needed.
+
+Installation
+============
+
+To install this module, simply follow the standard install process.
+
+Configuration
+=============
+
+No configuration is needed or possible.
+
+Usage
+=====
+
+* Inherit from ``base.kanban.abstract`` to add Kanban stage functionality to 
+  the child model:
+
+  .. code-block:: python
+
+    class MyModel(models.Model):
+        _name = 'my.model'
+        _inherit = 'base.kanban.abstract'
+        
+* Extend the provided base Kanban view (``base_kanban_abstract_view_kanban``) 
+  as needed by the child model. The base view has four ``name`` attributes 
+  intended to provide convenient XPath access to different parts of the Kanban 
+  card. They are ``card_dropdown_menu``, ``card_header``, ``card_body``, and 
+  ``card_footer``:
+
+  .. code-block:: xml
+
+    <record id="my_model_view_kanban" model="ir.ui.view">
+        <field name="name">My Model - Kanban View</field>
+        <field name="model">my.model</field>
```

Comment:
Add `name="mode">primary<` & a note indicating that this must be set in order for the proper inheritance. Per the [docs](https://www.odoo.com/documentation/10.0/reference/views.html):

> mode
>     inheritance mode, see Inheritance. If inherit_id is unset the mode can only be primary. If inherit_id is set, extension by default but can be explicitly set to primary